swagger.yaml

```diff
@@ -8530,7 +8530,7 @@ definitions:
       - 'hostname-unique'
       - 'hostname-valid'
       - 'belongs-to-machine-cidr'
-      - 'api-vip-connected'
+      - 'ignition-downloadable'
```

Comment:
@rollandf lgtm, we'll need just a fancy name for this validation...

Reply:
> @rollandf lgtm, we'll need just a fancy name for this validation...

Do we need a Jira to follow up?

Reply:
Not sure. It is only being used [here](https://github.com/openshift-assisted/assisted-ui-lib/blob/d660d51055dfc018edbd57f0a3298f9ea9cc037f/src/common/config/constants.ts#L177)